Source/Shaders/ModelExperimental/LightingStageFS.glsl

```diff
@@ -7,16 +7,75 @@ vec3 LINEARtoSRGB(vec3 linearIn)
     #endif 
 }
 
+#ifdef HAS_OUTLINES
```

Comment:
We're not setting this flag anywhere, should this be part of a later PR?

Reply:
ah yeah, this should be done later when we get to outlines for Cesium OSM buildings. I'll remove it for now.